config/env/dev.json

```diff
@@ -56,6 +56,7 @@
     "traceRatio": "@@METRICS_TRACE_RATIO",
     "exportIntervalMillis": "@@METRICS_EXPORT_INTERVAL_MS",
     "exportTimeoutMillis": "@@METRICS_EXPORT_TIMEOUT_MS",
+    "instanceIdentifier": "@@ECS_CONTAINER_METADATA_URI",
```

Comment:
I'm not familiar with the `@@` syntax. This will set the value to the corresponding env var?

Reply:
yes that's correct - so if present, it is going to set it from ECS_CONTAINER_METADATA_URI which is present on the api server instances.  It will not get set otherwise and won't be used as a label if its not present